daemon/cmd/health.go

```diff
@@ -92,7 +92,7 @@ func (d *Daemon) initHealth(spec *healthApi.Spec, cleaner *daemonCleanup, sysctl
 						d.mtuConfig,
 						d.bigTCPConfig,
 						d.endpointManager,
-						d.l7Proxy,
+						nil, // create health endpoint without l7 proxy
```

Comment:
why not removing this parameter alltogether from `health.LaunchAsEndpoint` and move the comment to the place where we call `endpoint.NewEndpointFromChangeModel` with `nil` as parameter?

Reply:
Good call, changed as suggested!

Reply:
thanks!